.github/workflows/build-docker-images-release.yml

```diff
@@ -41,7 +41,7 @@ jobs:
 
   version-cuda:
     name: "Latest Accelerate GPU [version]"
-    runs-on: [self-hosted, docker-gpu, multi-gpu, gcp]
+    runs-on: [self-hosted, intel-cpu, 8-cpu, ci]
```

Comment:
a bit strange that here we use cpu machine while the job name is GPU

Reply:
Fixed!

Reply:
i agree, but it was docker-cpu...

Reply:
it's for building image. so maybe no need GPU ?